src/test/java/org/threeten/extra/chrono/TestFrenchRepublicChronology.java

```diff
@@ -330,9 +330,7 @@ public void test_chronology_dateYearDay_badDate() {
     @Test
     public void test_isLeapYear_loop() {
         for (int year = -200; year < 200; year++) {
-            FrenchRepublicDate base = FrenchRepublicDate.of(year, 1, 1);
-            assertEquals(base.isLeapYear(), ((year+1) % 4) == 0);
-            assertEquals(FrenchRepublicChronology.INSTANCE.isLeapYear(year), ((year+1) % 4) == 0);
+            assertEquals(FrenchRepublicChronology.INSTANCE.isLeapYear(year), ((year + 1) & 3) == 0);
         }
```

Comment:
I think the idea here is testing exhaustively both the static and non static versions of isLeapYear().